sites/website/src/docs/getting-started/fast-element.md

```diff
@@ -0,0 +1,310 @@
+---
+id: fast-element
+title: FASTElement
+sidebar_label: FASTElement
+keywords:
+  - FASTElement
+  - web components
+---
+
+# FASTElement
+
+The `FASTElement` class can be extended from for your custom component logic.
+
+## Attribute Bindings
+
+Attributes are defined using the `@attr` decorator.
+
+**Example:**
+```ts
+import { FASTElement, attr } from '@microsoft/fast-element';
+
+export class MyElement extends FASTElement {
+  @attr
+  foo: string;
+}
+```
+
+HTML file:
+```html
+<my-element foo="Hello"></my-element>
+```
+
+An `@attr` can take a configuration with the following options:
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| attribute | The attribute name that is reflected in the DOM, this can be specified in cases where a different string is preferred. | `string` | The class property converted to lowercase |
+| mode | If the attribute is a boolean and the mode is set to "boolean" this allows `FASTElement` to add/remove the element from the DOM. The "fromView" behavior only updates the property value based on changes in the DOM, but does not reflect property changes back. | `"reflect" \| "boolean" \| "fromView"` | "reflect" |
```

Comment:
Should we note that this mimics the standard html behavior for Boolean attributes, its presence as true and its absence as false?